### PR TITLE
Add CONTRIBUTING.md

### LINKED ISSUE #16: Add CONTRIBUTING.md
This can and should be very simple to start.

Previous text:

## Contributing
Contributions are very welcome. Though we ask that you open an issue or pull request early in the process (before writing code) so we can discuss solutions and additions before you start spending time on implementing them. There are some specific areas where we would be extra happy to receive contributions in.

- Binary size reduction
- Runtime performance
- Ensure build / test environment works well on non macOS platforms
- Alternate layout systems (grid layout perhaps?)
- Web compatibility tests
- RTL support
- Platform bindings
- API improvements
- Documentation & Examples

### PR DESCRIPTION
**Pull request by alice-i-cecile**
Fixes #16.

Existing information about contributing has been removed from the README.

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+The basic process is simple:
+
+1. Pick an issue, or file a new one.
+2. Comment in the issue that you plan to tackle it, and the team will assign the task to you.
+3. Submit a PR.
+4. Respond to feedback from reviewers and make sure CI passes.
+
+PR review is completely open: if you'd like to help us comment on, test and approve PRs.
+
+Questions of "how do I use this library" belong in GitHub Discussions, although those should typically spawn new issues to improve our docs and examples.
```
Comment (TimJentzsch) [inline CONTRIBUTING.md:15]:
```suggestion
Questions like "how do I use this library?" belong in [GitHub Discussions](https://github.com/DioxusLabs/stretch/discussions/new), although those should typically spawn new issues to improve our docs and examples.
```

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,17 @@
+# Contributing
+
+This is a cross-team project, aiming to build solid foundations for Rust UI libraries of all sorts.
+New contributions are extremely welcome!
+
+The basic process is simple:
+
+1. Pick an issue, or file a new one.
```
Comment (TimJentzsch) [inline CONTRIBUTING.md:8]:
```suggestion
1. Pick an [issue](https://github.com/DioxusLabs/stretch/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22), or [file a new one](https://github.com/DioxusLabs/stretch/issues/new).
```

File: README.md [hunk 3]
```diff
@@ -131,19 +131,6 @@ const layout = node.computeLayout();
 console.log(layout.width, layout.height);
 ```
 
-## Contributing
```
Comment (TimJentzsch) [inline README.md:134]: Should we add a reference to the CONTRIBUTING.md here or do you want to keep it completely out of the README?

**Review by TimJentzsch — COMMENTED** [review]
I added some minor comments to add links for quick access and phrasing suggestions.

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+New contributions are extremely welcome!
+
+The basic process is simple:
+
+1. Pick an issue, or file a new one.
+2. Comment in the issue that you plan to tackle it, and the team will assign the task to you.
+3. Submit a PR.
+4. Respond to feedback from reviewers and make sure CI passes.
+
+PR review is completely open: if you'd like to help us comment on, test and approve PRs.
```
[the remark this answers is not in the document]
Comment (alice-i-cecile) [reply CONTRIBUTING.md:13]:
```suggestion
The PR review process is completely open:  help us by commenting on, testing and approving PRs.
```

File: README.md (hunk 3 above)
Comment (alice-i-cecile) [reply README.md:144]: I don't think it's worth it. This will let us include the README as the crate-level docs, which is very convenient.

**Pull request merged**